scripts/integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -ex
+set -o pipefail
+
+if [ ! -f genfiles/server.key ]; then
+	./scripts/prepare_server.sh -f
+fi
+
+docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
+trap "docker-compose down" INT EXIT
+
+set +e
+wget -t 60 --spider --retry-connrefused --waitretry=1  0.0.0.0:443 || sleep 2
```

Comment:
`-t 60` means make 60 retries, but I suspect you wanted to retry for 60 seconds. 

```
  -t,  --tries=NUMBER              set number of retries to NUMBER (0 unlimits)
  -T,  --timeout=SECONDS           set all timeout values to SECONDS
```